cmd/init.go

```diff
@@ -49,10 +48,10 @@ func runInit(cmd *cobra.Command, args []string) error {
 }
 
 type initConfig struct {
-	// Name of the service in DNS-compatible format (ex myfunc.example.com)
+	// Name of the function.
 	Name string
 
-	// Path to files on disk.  Defaults to current working directory.
+	// Absolute path to project on disk.
```

Comment:
While this is still yet to be discussed, it may be worth mentioning:  a "project" is likely to be a _grouping_ of Functions.  Such as the entire `example.com` directory, with a new config file at that parent level.  Our code at persent is just for Functions at this point, not really projects... yet!